docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1003,31 +1005,146 @@ async function getUserMedia(constraints = {audio: true, video: true}) {
   if (!meeting) {
     console.log('MeetingControls#getUserMedia() :: no valid meeting object!');
 
-    return Promise.reject(new Error('No valid meeting object.'));
+    throw new Error('No valid meeting object.');
+  }
+
+  if (localMedia.microphoneStream) {
+    console.log('MeetingControls#getUserMedia() :: microphone stream is already available');
+  }
+  else if (constraints.audio) {
+    await loadMicrophone(constraints.audio);
+  }
+
+  if (localMedia.cameraStream) {
+    console.log('MeetingControls#getUserMedia() :: camera stream is already available');
+  }
+  else if (constraints.video) {
+    await loadCamera(constraints.video);
+  }
+
+  console.log('MeetingControls#getUserMedia() :: following local streams are ready to use:', localMedia.microphoneStream, localMedia.cameraStream);
+}
+
+async function loadCamera(constraints) {
+  try {
+    console.log('MeetingControls#loadCamera() :: using webrtc-core local camera stream');
+    const videoConstraints = {...localMedia.videoConstraints[localVideoQuality[localResolutionInp.value]], ...constraints};
+
+    console.log('MeetingControls#loadCamera() :: getting camera stream with constraints: ', videoConstraints);
+    localMedia.cameraStream = await webex.meetings.mediaHelpers.createCameraStream(videoConstraints);
+
+    meetingStreamsLocalVideo.srcObject = localMedia.cameraStream.outputStream;
+    toggleVbgBtn.innerText = "Enable VBG";
+    loadCameraBtn.disabled = true;
+    console.log('MeetingControls#loadCamera() :: Successfully got camera stream:', localMedia.cameraStream);
+  }
+  catch (e) {
+    console.log('MeetingControls#loadCamera() :: Error getting camera stream!');
+    throw e;
+  }
+}
+
+async function handleVbg() {
```

Comment:
when testing I've found the following problems:
- the modes image and video don't work at all, only blur works - is that expected?
- pressing toggle video does nothing if it's pressed before I join a meeting - there is no feedback to the user that you have disabled the video

Reply:
1. It will work but for local testing (till this PR is not merged), we need to update the path for image and video urls to point to local assets folder path. I have tested locally, both works fine.
image: `https://localhost:8000/api/assets/vbg_image.jpg`
video: `https://localhost:8000/api/assets/clouds.5b57454a.mp4`

2. toggling of video needs a meeting instance so it won't work before joining a meeting. Shall we disable the toggle button till we call addMedia or any other suggestion?

Reply:
2. The whole point of having webrtc-core classes like LocalCameraStream and LocalMicrophoneStream is that now you can mute/unmute them outside any meetings, by calling setMuted() on them, so it would be wrong for the sample app to only allow it during a meeting

Reply:
Updated the code to allow mute/unmute of audio/video without a meeting instance as well.